src/Components/CanvasComponent.tsx

```diff
@@ -245,7 +253,7 @@ class CanvasComponent extends React.Component {
           >
             <FontAwesomeIcon icon={faDownload}/>
           </button>
-          <input type="color" onChange={this.handleChangeComplete}/>
+          <input type="color" ref={this.colorInput} onChange={this.handleChangeComplete}/>
```

Comment:
Set `value` to `this.state.color`. That way, you won't have to explicitly set the value in the handler.